Source/WarpX.H

```diff
@@ -273,6 +275,30 @@ public:
     const amrex::MultiFab& getEfield_avg_fp (int lev, int direction) {return *Efield_avg_fp[lev][direction];}
     const amrex::MultiFab& getBfield_avg_fp (int lev, int direction) {return *Bfield_avg_fp[lev][direction];}
 
+    const amrex::MultiFab& getGatheredRho_fp (int lev) {
+
```

Comment:
Extraneous newline here? At least it's not consistent with below functions you added.